superset/views/csv_import.py

```diff
@@ -145,22 +150,31 @@ def add(self) -> Response:
             self._fill_table(form_data, table, csv_filename)
         except (
             NameNotAllowed,
-            ValueError,
+            ConversionException,
+            DatabaseAlreadyExist,
             SchemaNotAllowedCsvUpload,
             NoResultFound,
             MultipleResultsFound,
+            SchemaException,
+            FileSaveException,
             NoUsernameSuppliedException,
             NoPasswordSuppliedException,
-            OSError,
         ) as e:
+            LOGGER.exception(f"Failed prepare CSV import {e.orig}")
             STATS_LOGGER.incr("failed_csv_upload")
             return json_error_response(e.args[0], status=BAD_REQUEST)
         except (DatabaseCreationException, TableCreationException) as e:
+            LOGGER.exception(f"Failed import CSV {e.orig}")
             STATS_LOGGER.incr("failed_csv_upload")
             if NEW_DATABASE_ID == database_id:
                 self._remove_database(database, db_flavor)
             return json_error_response(e.args[0], status=BAD_REQUEST)
-        except (Exception, DatabaseDeletionException) as e:
+        except DatabaseDeletionException as e:
+            LOGGER.exception(f"Failed delete Database {e.orig}")
```

Comment:
Failed to deleted Database
Error occured while trying to delete database
Database deletion failed